src/accessors/external_accessor.ts

```diff
@@ -191,15 +186,15 @@ export class ExternalAccessor implements IConsumerApiAccessor {
 
   private _buildStartProcessInstanceUrl(processModelId: string,
                                         startEventId: string,
-                                        startCallbackType: StartCallbackType,
+                                        startCallbackType: DataModels.ProcessModels.StartCallbackType,
                                         endEventId: string): string {
     let url: string = restSettings.paths.startProcessInstance
       .replace(restSettings.params.processModelId, processModelId)
       .replace(restSettings.params.startEventId, startEventId);
 
     url = `${url}?start_callback_type=${startCallbackType}`;
 
-    if (startCallbackType === StartCallbackType.CallbackOnEndEventReached) {
+    if (startCallbackType === DataModels.ProcessModels.StartCallbackType.CallbackOnEndEventReached) {
```

Comment:
Extract Business Rule (Maybe you can find a better name)

```suggestion
    const startCallBackIsOnEndEventReached: boolean = startCallbackType === DataModels.ProcessModels.StartCallbackType.CallbackOnEndEventReached;
    if (startCallBackIsOnEndEventReached) {
```